packages/custom-esbuild/package.json

```diff
@@ -41,15 +41,15 @@
   "builders": "builders.json",
   "dependencies": {
     "@angular-builders/common": "workspace:*",
-    "@angular-devkit/architect": ">=0.2000.0 < 0.2100.0",
-    "@angular-devkit/core": "^20.0.0",
-    "@angular/build": "^20.0.0"
+    "@angular-devkit/architect": ">=0.2001.0 < 0.2100.0",
+    "@angular-devkit/core": "^20.1.0",
+    "@angular/build": "^20.1.0"
   },
   "peerDependencies": {
-    "@angular/compiler-cli": "^20.0.0"
+    "@angular/compiler-cli": "^20.1.0"
   },
   "devDependencies": {
-    "esbuild": "0.25.1",
+    "esbuild": "~0.25.1",
```

Comment:
ESBuild version here must be compatible with the version used in `@angular/build` otherwise there will be a TS error.